src/WebJobs.Script/Config/ScaleOptionsSetup.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.WebJobs.Host.Scale;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.Azure.WebJobs.Script.Configuration
+{
+    internal class ScaleOptionsSetup : IConfigureOptions<ScaleOptions>
+    {
+        private readonly IConfiguration _configuration;
+        private readonly IEnvironment _environment;
+
+        public ScaleOptionsSetup(IConfiguration configuration, IEnvironment environment)
+        {
+            _configuration = configuration;
+            _environment = environment;
+        }
+
+        public void Configure(ScaleOptions options)
+        {
+            var jobHostSection = _configuration.GetSection(ConfigurationSectionNames.JobHost);
```

Comment:
SDK is already binding these options. Do we need this? How does it interact with the other bind?

Reply:
SDK does not configure the options - we can pass the options into `ConfigureWebJobsScale`. `ConfigureWebJobsScale `is not called from the function host so we need a way to configure it the function host. I moved the code to setup class from [here](https://github.com/Azure/azure-functions-host/pull/9131/files/#diff-0520ea7869aafe4adf4e46cee524515169549321187cc8ecf473409fe070b53cL309).

Reply:
I'm talking about the code [here](https://github.com/Azure/azure-webjobs-sdk/blob/55f3d9a7c8e0f38c9ad31b94585c9d395eeb7f57/src/Microsoft.Azure.WebJobs.Host/Hosting/WebJobsServiceCollectionExtensions.cs#L196) in the SDK that binds these same options.

Reply:
You are right, I missed the binding.